### DISCUSSION TITLE
Crab steering on vehicles who have that available

### DISCUSSION THREAD

**Discussion by DickTimmerman** [Courseplay/Courseplay_FS22#3224] [Ideas]
Hi, woul'd it be possible or is it already possible (i can't find it) to use crab steering on those vehicles who are able to do that?
Woul'd make the experience with CP even bigger.

**Comment by Tensuko** [comment]
Not planed, as it would need complete control over it + change direection and overloading will be even more complicated.
There is no real benefit in it, only visual and for that the effert is rly high... Maybe in the far future.
